src/styles.scss

```diff
@@ -169,12 +169,16 @@ $accent-success: #88b42d;
 .text-h6 {
   font-size: clamp(1rem, 0.25vw + 0.94rem, 1.125rem);
 }
+.text-h7 {
```

Comment:
do czego jest używany text-h7? h6 to najmniejszy nagłówek, czy może inna nazwa byłaby lepsza?

Reply:
To była pozostałość z testowania czy jest sens stworzyć karty o mniejszym rozmiarze. Usunąłem.